bindings/rust/wasmedge-sys/src/io.rs

```diff
@@ -0,0 +1,2461 @@
+use std::marker::PhantomData;
+
+use crate::Value;
+
+fn match_value(s: &str) -> Value {
+    match s {
+        "i8" => Value::I32(0),
+        "u8" => Value::I32(0),
+        "i16" => Value::I32(0),
+        "u16" => Value::I32(0),
+        "i32" => Value::I32(0),
+        "u32" => Value::I64(0),
+        "i64" => Value::I64(0),
+        "f32" => Value::F32(0.),
+        "f64" => Value::F64(0.),
+        "u128" => Value::V128(0),
+        _ => panic!("unsupport type for WasmFnIO"),
+    }
+}
+pub trait WasmFnIO {
+    fn parameters() -> Vec<Value>
+    where
+        Self: Sized,
+    {
+        Vec::new()
+    }
+}
+
+pub struct I1<O1> {
+    phantom_1: PhantomData<O1>,
+}
+
+impl<O1> WasmFnIO for I1<O1> {
+    fn parameters() -> Vec<Value> {
+        vec![match_value(std::any::type_name::<O1>())]
+    }
+}
+
+pub struct I2<O1, O2> {
+    phantom_1: PhantomData<O1>,
+    phantom_2: PhantomData<O2>,
+}
+
+impl<O1, O2> WasmFnIO for I2<O1, O2> {
+    fn parameters() -> Vec<Value> {
+        vec![
+            match_value(std::any::type_name::<O1>()),
+            match_value(std::any::type_name::<O2>()),
+        ]
+    }
+}
```

Comment:
This part should be generated by writing a macro in the future, and when `std::any::type_name` supports `const fn`, it should also be possible to replace it with a constant generic type.

Reply:
Ok, I can try to modify it